abstractions/model_free/sac/train.py

```diff
@@ -105,7 +105,9 @@ def episode_loop(env, test_env, agent, replay_buffer, args, writer):
             else:
                 clipped_reward = reward
 
-            mask = 1 if steps == env._max_episode_steps else not done # pylint: disable=protected-access
+            # mask = 1 if steps == env._max_episode_steps else not done
+            # pylint: disable=protected-access
+            mask = not done
```

Comment:
Why did you decide not to limit the episode length anymore?

Reply:
I wasn't sure of the validity of that approach. Also, our wrapped envs don't have the _max_episode_steps so I wanted to check with you first.